idunn/api/poi.py

```diff
@@ -73,7 +78,7 @@ class POI(types.Type):
     subclass_name = validators.String(allow_null=True)
     geometry = validators.Object(allow_null=True)
     address = validators.Object(allow_null=True)
-    blocks = validators.Array(items=BLOCKS_ORDER)
+    blocks = validators.Array()
```

Comment:
why do you removed this ?
It's not a big problem as the schema is currently broken, but in the futur, we'll want to reactive this